### PR TITLE
Feature Combobox, updated targetframeworks

### LINKED ISSUE #18: MultiSelect (checkbox) support

### PR DESCRIPTION
**Pull request by m-nab**
## Description
Added a ComboBox prompt, also updated the targetframework to include the newly released .net core 3.0

Could perhaps close #18 

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## How Has This Been Tested?
Has been extensively tested on my own configuration.

**Test Configuration**:
* Hardware: Windows 7 Ultimate x64
* Toolchain: CMD

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: Sharprompt/Sharprompt.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+     <TargetFrameworks>netstandard2.0;netstandard2.1;</TargetFrameworks>
```
Comment (shibayan) [inline Sharprompt/Sharprompt.csproj]: .NET Standard 2.1 is not necessary, please return to 2.0.

File: Sharprompt/Prompt.cs [hunk 2]
```diff
@@ -24,5 +24,9 @@ public static T Select<T>(string message, IEnumerable<T> items, object defaultVa
         {
             return new Select<T>(message, items, defaultValue, pageSize, valueSelector ?? (x => x.ToString())).Start();
         }
+        public static IEnumerable<T> ComboBox<T>(string message, IEnumerable<T> items, object defaultValue = null, int pageSize = 10, int limit = -1, int min = 1, Func<T, string> valueSelector = null)
+        {
+            return new ComboBox<T>(message, items, limit, min, pageSize, valueSelector ?? (x => x.ToString())).Start();
+        }
```
Comment (shibayan) [inline Sharprompt/Prompt.cs]: In general, ComboBox is almost the same as DropDownList and does not represent a multi-selectable GUI. Change to MultiSelect.

File: Sharprompt/ComboBox.cs [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+            _baseOptions = options.Select(x => new Option(valueSelector(x), x, IsOptionEnabled(x))).ToArray();
+        }
+
+        private readonly string _message;
+        private readonly int _limit;
+        private readonly int _min;
+        private readonly int _pageSize;
+        private readonly Func<string, string, bool> _filtering;
+        private readonly IReadOnlyList<Option> _baseOptions;
+        private const string ARROW_RIGHT = "\u0010";
```
Comment (shibayan) [inline Sharprompt/ComboBox.cs]: Use Pascal Case for constants.

File: Sharprompt/ComboBox.cs [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+            renderer.WriteMessage(model.Message);
+            renderer.Write(model.Filter);
+            if (_showConfirm)
+            {
+                renderer.Write(" Press Tab to confirm", ConsoleColor.Cyan);
+            }
+
+            for (int i = 0; i < model.Options.Count; i++)
+            {
+                Option currentOption = model.Options.ElementAt(i);
```
Comment (shibayan) [inline Sharprompt/ComboBox.cs]: Avoid using ElementAt and use indexer.

File: Example/Example.csproj [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
```
Comment (shibayan) [inline Example/Example.csproj]: Please return it because it is not related to this feature.

**Comment by m-nab** [comment]
Thank you for your feedback!. I will create a new feature-request, since the rename impacts the name of the branch.

**Pull request closed without merging**